README.md

```diff
@@ -79,6 +80,9 @@ To illustrate, `bin/rails console` will run the console in the docker container
 - [ExceptionHunter](https://github.com/rootstrap/exception_hunter) for exception tracking
 - [Factory Bot](https://github.com/thoughtbot/factory_bot) for testing data
 - [Faker](https://github.com/stympy/faker) for generating test data
+- [flipper](https://github.com/jnunemaker/flipper) for feature flag support
+- [flipper-active_record](https://github.com/jnunemaker/flipper) for Flipper active_record support
+- [flipper-ui](https://github.com/jnunemaker/flipper) for UI-based feature flag management
```

Comment:
maybe in this place of the docs we can just say

```
- [Flipper](https://github.com/jnunemaker/flipper) for feature flag support
```

the activerecord and UI things can be implicit I guess?